src/main/java/br/com/swconsultoria/nfe/util/StubUtil.java

```diff
@@ -0,0 +1,32 @@
+package br.com.swconsultoria.nfe.util;
+
+import br.com.swconsultoria.certificado.Certificado;
+import br.com.swconsultoria.certificado.CertificadoService;
+import br.com.swconsultoria.certificado.exception.CertificadoException;
+import br.com.swconsultoria.nfe.dom.ConfiguracoesNfe;
+import org.apache.axis2.client.Stub;
+import org.apache.axis2.transport.http.HTTPConstants;
+import org.apache.commons.httpclient.HttpClient;
+
+public class StubUtil {
+
+    /**
+     *
+     * @param stub {@link org.apache.axis2.client.Stub} que terá a conexão configurada para o {@link org.apache.commons.httpclient.HttpClient}
+     *             específico do certificado.
+     * @param configuracoesNfe {@link br.com.swconsultoria.nfe.dom.ConfiguracoesNfe} config que possui o {@link br.com.swconsultoria.certificado.Certificado}
+     *                         que será configurado no HttpClient.
+     * @param url {@link String} url que será executada.
+     * @throws CertificadoException
+     */
+    public static void configuraHttpClient(Stub stub, ConfiguracoesNfe configuracoesNfe, String url) throws CertificadoException {
+        Certificado certificado = configuracoesNfe.getCertificado();
```

Comment:
Uma dúvida sobre esse, segundo esse cenario ele sempre será executado certo? 
Aqui não deveriamos verificar com a nova variavel "isModoMultithreading"?

Reply:
Verdade.

Aqui seria interessante ajustar o Java Doc para dizer que faz apenas se for `isModoMultithreading` e fazer o if.